src/Atmos/Model/turbulence.jl

```diff
@@ -23,6 +23,41 @@ end
 function gradvariables!(::TurbulenceClosure, transform::Vars, state::Vars, aux::Vars, t::Real)
 end
 
+struct NoViscosity <: TurbulenceClosure end
```

Comment:
Thanks for setting this test up! Could we use the current setup with `ConstantViscosityWithDivergence(zero(DFloat))` to get us the same physical result of having no viscosity ? diffusive.\rho\tau is defined at the `AtmosModel` top level

Reply:
The problem I see with the `ConstantViscosityWithDivergence(zero(DFloat))` approach is that if you want to look at performance you will still call all the viscous kernels, which are not needed in this case. When we start splitting the calculation into substeps, we certainly want to be able to turn these kernel calls off.

Reply:
Another bonus is that being able to turn viscosity off completely can sometimes help with debugging. If my current changes are too intrusive then there is another approach. You could disable viscosity at the top level of `AtmosModel` by defining methods like

```
flux_diffusive!(m::AtmosModel{O,RS,T}, _...) where {O,RS,T<:NoViscosity} = nothing
```
Then other parts of `AtmosModel`, including the location of `ρτ`, can remain as they were before my changes. Would that  be better ?

@charleskawczynski

Reply:
> The problem I see with the `ConstantViscosityWithDivergence(zero(DFloat))` approach is that if you want to look at performance you will still call all the viscous kernels, which are not needed in this case

That's a great point, I definitely see the motivation for this.

@mwarusz The only issue I have with the example you suggested is that every function signature of this flavor may have to change every time a new component is added to `AtmosModel`, which is kind of a pain. Might we instead just add an argument and use something like 

```
function flux_diffusive!(m::AtmosModel, _...)
  flux_diffusive!(m, ..., m.turbulence)
end
function flux_diffusive!(m::AtmosModel, _..., ::NoViscosity)
...
end
```

But this might lead to very nested calls. @simonbyrne  what do you think?